plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/GcpLocalRunTab.java

```diff
@@ -413,9 +417,14 @@ Path getServiceAccountKeyPath() {
     
     // can't use colons in filenames on Windows
     filename = filename.replace(':', '.');
-    
-    String configurationLocation = Platform.getConfigurationLocation().getURL().getPath();
-    return Paths.get(configurationLocation + "/com.google.cloud.tools.eclipse/" + filename);
+
+    try {
+      Path configurationLocation = Paths.get(Platform.getConfigurationLocation().getURL().toURI());
+      return configurationLocation.resolve("com.google.cloud.tools.eclipse").resolve(filename);
+    } catch (URISyntaxException e) {
+      logger.log(Level.SEVERE, "Cannot convert configuration location into URI", e); //$NON-NLS-1$
+      return null;
```

Comment:
Since this should never happen, is it worth either marking this as a thrown exception, or wrapping it into a `CoreException`?

Reply:
I don't feel like making an exception publically visible to callers, specifically because it is never meant to happen.